packages/docs/docs/getting-started/faqs.md

```diff
@@ -0,0 +1,28 @@
+---
+
+title: FAQs
+---
+
+# ❓FAQs
+
+What is Unirep?
```

Comment:
you can highlight the topic by 
e.g.
```
## What is Unirep?
```
preview:
## What is Unirep?
or 
```
**What is Unirep?**
```
preview:
**What is Unirep?**